bool activate # True to start checking for collisions, False for standby mode.
---
bool success